third_party/move/move-compiler-v2/tests/checking/typing/block_single_expr_invalid.exp

```diff
@@ -1,6 +1,6 @@
 
 Diagnostics:
-error: expected `bool` but found `integer` in constant
+error: `integer` not compatible with `bool` in constant
```

Comment:
error: expression of type `integer` not compatible with declared type `bool` in constant.